mediapipe/calculators/ovms/openvinomodelserversessioncalculator.cc

```diff
@@ -154,10 +154,11 @@ class OpenVINOModelServerSessionCalculator : public CalculatorBase {
             bool isServerReady = false;
             OVMS_ServerNew(&cserver);
 
-            if (triedToStartOVMS){
-                ASSERT_CAPI_STATUS_NULL(OVMS_ServerReady(cserver, &isServerReady));
+            ASSERT_CAPI_STATUS_NULL(OVMS_ServerReady(cserver, &isServerReady));
+
+            if (triedToStartOVMS) {
                 RET_CHECK(isServerReady);
-            } else {
+            } else if (!isServerReady) {
```

Comment:
I think this is incomplete fix. OVMS will respond that it is ready only when it finishes loading whole config.json. So it will still claim that it is not ready during loading of 2nd graph. Here we could check for liveness -> it will tell that we are inside OVMS server, not inside MP app.